app/modules/sightings/resources.py

```diff
@@ -476,20 +480,26 @@ def patch(self, args, sighting):
                 new_version = rdata['result'].get('version', None)
                 if new_version is not None:
                     sighting.version = new_version
-                with db.session.begin():
-                    db.session.merge(sighting)
+                    context = api.commit_or_abort(
+                        db.session,
+                        default_error_message='Failed to update Sighting version.',
+                    )
+                    with context:
+                        db.session.merge(sighting)
+            return rdata
```

Comment:
Re other comment, why not just return rdata['result']?

Reply:
again, i have just been following the format we have always expected from wbia.   it is a side-effect of edm mimicking wbia.  houston echoing edm.